src/Workers/net-worker/stellar-network.ts

```diff
@@ -92,7 +92,14 @@ export async function checkHorizonOrFailover(primaryHorizonURL: string, secondar
   try {
     const primaryResponse = await fetch(primaryHorizonURL)
     if (primaryResponse.ok) {
-      return primaryHorizonURL
+      // fetch dynamic data to check database access
+      const accountResponse = await fetch(
```

Comment:
Two questions:
1. Why fetch the index route first and only then fetch the account? Doesn't seem to add value to the health check, but will now take twice as long until the health check returns a result.
2. What account is GAIH… (comment would be helpful 😉)? And will it likely always be available on both pubnet and testnet?

Reply:
You're right, I somehow didn't realise the redundancy of both fetches. I changed it and added another comment that indicates that the account id belongs to the friendbot.